src/Donors/resources/admin/components/DonorDetailsPage.tsx

```diff
@@ -0,0 +1,11 @@
+import {__} from '@wordpress/i18n';
+
+const DonorDetailsPage = () => {
```

Comment:
Does this need the donorId?

Reply:
That's part of the actual implementation of the page, that will be in another PR.